### PR TITLE
Bug between shift and worker 

### LINKED ISSUE #73: Bug found worker listed as shift taker but worker's page shows no shifts taken
Demo'd at the team sync. When a worker is listed as the taker of a shift, that shift does not appear on the worker's page of shifts.

Checked the databases, it looks like shifts update worker_id field with user's id instead of worker's id 
Screen Shot 2021-04-29 at 4 02 02 PM

### PR DESCRIPTION
**Pull request by ncbui**
## What type of PR is this? (check all applicable)

- [] Feature
- [] Refactor
- [] Bug Fix
- [] Optimization
- [] Documentation Update
- [] Other (describe: )

## Description of what PR does
This PR fixes a bug demo'd in today's team sync. Worker id called when workers take and drop shifts. Worker's pages now look for shifts associated with their worker id (will have to use psql to check)

## Related PRs and/or Issues (if any)
- Closes #73 
- Unblocks #45 #52

## QA Instructions, Screenshots, Recordings
Navigate to http://127.0.0.1:3000/
While logged in as a worker, ensure that worker's shifts show on their page. Sign up for a shift to make sure worker's id is stored in database instead of user id

Note: does not alter shifts that are currently in database

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed _(please include reasons for why tests aren't needed)_
- [ ] no, because I need help
- [] no, they will be added later (please create an Issue for it)

## Added to documentation?

- [ ] Yes, project README
- [x ] No documentation needed

### CODE REVIEW

**Comment by GabrielleEaston** [comment]
@ncbui Great debugging skills!

**Review by GabrielleEaston — APPROVED** [review]

**Comment by msarit** [comment]
resolve conflicts, @ncbui then I will merge 😄

File: app/controllers/shifts_controller.rb [hunk 1]
```diff
@@ -63,14 +63,14 @@ def take
       flash[:danger] = "This Shift is already taken!"
       redirect_to shifts_path
     else
-      @shift.update_columns(worker_id: current_user.id, shift_open: false)
+      @shift.update_columns(worker_id: current_user.worker_account.id, shift_open: false)
```
Comment (msarit) [inline app/controllers/shifts_controller.rb:66]: perfect debugging! great job!

**Review by msarit — APPROVED** [review]

**Comment by ncbui** [comment]
Thanks @msarit ! Have resolved the conflict

**Pull request merged**